### PR TITLE
General edits to resampling method

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: source/resampling_method.Rmd [hunk 1]
```diff
@@ -485,22 +523,17 @@ given day. And we store that result in a location that we decide to call `kk`,
...
 that result on the screen.
 
-Please don't worry about the details of how each of these commands works --- we
+Don't worry too much about the details of how each of these commands works --- we
 will cover those details gradually, over the next few chapters.  But, we hope
-that you can see in principle, how each of the operations that the computer
+that you can see, in principle, how each of the operations that the computer
 carries out are analogous to the operations that you yourself executed when you
-solved this problem using a ten-sided spinner or a random-number table. This is
+solved this problem using a ten-sided die. This is
```
Comment (matthew-brett) [inline source/resampling_method.Rmd:530]:
```suggestion
solved this problem using the equivalent of a ten-sided die. This is
```

File: source/resampling_method.Rmd [hunk 2]
```diff
@@ -512,14 +545,16 @@ professional mathematician, it may take you quite a while to arrive at
 the correct formula---considerable hard thinking, and perhaps some
 digging in textbooks. More important than the labor, however, is that
 you may come up with the wrong formula, and hence obtain the wrong
-answer. Most students who have had a standard course in probability and
+answer. And how would you know if you were wrong?
+
+Most students who have had a standard course in probability and
 statistics are quick to tell you that it is not easy to find the correct
 formula, even immediately after finishing a course (or several courses)
 on the subject. After leaving school, it is harder still to choose the
```
Comment (matthew-brett) [inline source/resampling_method.Rmd:553]:
```suggestion
on the subject. After leaving school or university it is harder still to choose the
```

File: source/resampling_method.Rmd [hunk 3]
```diff
@@ -512,14 +545,16 @@ professional mathematician, it may take you quite a while to arrive at
...
 right formula. Even many people who have taught statistics at the
 university level (including this writer) must look at a book to get the
 correct formula for a problem as simple as the ambulances, and then we are
-not always sure of the right answer. This is the grave disadvantage of
+still often unsure of the right answer. This is the grave disadvantage of
```
Comment (matthew-brett) [inline source/resampling_method.Rmd:557]:
```suggestion
often still not sure we have the right answer. This is the grave disadvantage of
```

**Review by matthew-brett — COMMENTED** [review]
Struggling still with the explanation of the for loop.  Can we get away without explaining, for now, and just give them the general idea?  I think the explanation will be frightening.

**Comment by stefanv** [comment]
> Struggling still with the explanation of the for loop. Can we get away without explaining, for now, and just give them the general idea? I think the explanation will be frightening.

I added a callout box to explain for loops only for students who want to read it (or who need to read it).  We should make it clear somewhere in the beginning of the book that callouts are optional (but often helpful).

**Comment by matthew-brett** [comment]
Merging to edit ...

**Pull request merged**